src/framework/preferences/js/Panels.js

```diff
@@ -692,6 +692,93 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     });
 
+    /********************************************************
+     * A base grade for styled radio Button adjuster panels *
+     ********************************************************/
+
+    fluid.defaults("fluid.prefs.panel.styledRadioButtonAdjuster", {
+        gradeNames: ["fluid.prefs.panel"],
+        mergePolicy: {
+            "controlValues.theme": "replace",
```

Comment:
@cindyli sort of yes, as the radio buttons would correspond to some sort of previewable thing. Do you have suggestions for a rename both for this and the component itself?